src/main/target/SPRACINGF3MINI/target.h

```diff
@@ -92,8 +92,13 @@
 #define USE_UART1
 #define USE_UART2
 #define USE_UART3
+
+#ifdef TINYBEEF3
```

Comment:
Why are you changing this? This is a brushed micro board with no pins except for the 4 motor outputs available, so having softserial compiled in doesn't make sense really.